composer.json

```diff
@@ -154,7 +154,7 @@
         "symfony/yaml": "^5.4 || ^6.0",
         "terminal42/escargot": "^1.4.1",
         "terminal42/service-annotation-bundle": "^1.1",
-        "toflar/psr6-symfony-http-cache-store": "^3.0",
+        "toflar/psr6-symfony-http-cache-store": "^3.0 || ^4.0",
```

Comment:
@Toflar is there something I need to check with this update?

Reply:
It's just because compatibility with both is not possible, see https://github.com/Toflar/psr6-symfony-http-cache-store#supported-branches